evol/logger.py

```diff
@@ -39,12 +39,19 @@ def __init__(self, target=None, stdout=False, fmt='%(asctime)s,%(message)s'):
                 self.logger.addHandler(stream_handler)
         self.logger.setLevel(level=logging.INFO)
 
+    @staticmethod
+    def check_population(population):
+        if any([i.fitness is None for i in population]):
```

Comment:
The islands PR contains an `is_evaluated` property on the population. That would be nice to use here.

Reply:
Sure. Is it easier to merge this and then the Islands or the other way around?

Reply:
As long as we fix it in the end 😊

Reply:
agreed https://github.com/godatadriven/evol/issues/141